vowpalwabbit/cbify.cc

```diff
@@ -467,6 +477,7 @@ base_learner* cbifyldf_setup(options_i& options, vw& all)
 
   data->app_seed = uniform_hash("vw", 2, 0);
   data->all = &all;
+  data->use_adf=true;
```

Comment:
Out of curiosity, what is the impact of not setting this?

Reply:
The finish() doesn't work properly.